lib/Zonemaster/Backend/RPCAPI.pm

```diff
@@ -268,7 +268,7 @@ sub _check_domain {
         }
     }
 
-    if ( $domain !~ m/^[\-a-zA-Z0-9\.\_]+$/ ) {
+    if ( $domain !~ m{^[a-z0-9_./-]+$}i ) {
```

Comment:
Have you considered calling untaint_ldh_domain() here instead?

Reply:
No, my approach was to make minimal update, and I did not even think of it before you mentioned it.

&untaint_ldh_domain() also tests the length of the domain namn, which I guess should not be tested here. That should rather be handled by test case code. Independent, that would go beyond the purpose of this PR.